src/ate/test_programs/cp.cc

```diff
@@ -230,8 +230,9 @@ int main(int argc, char **argv) {
   dut->DutFpgaLoadBitstream(fpga_bitstream_path);
   dut->DutLoadSramElf(openocd_path, sram_elf_path, /*wait_for_done=*/false,
                       /*timeout_ms=*/1000);
-  dut->DutTxCpProvisioningData(spi_frame.payload, spi_frame.cursor,
-                               /*timeout_ms=*/1000);
+  dut->DutConsoleTx(std::string("Waiting for CP provisioning data ..."),
```

Comment:
Do you need to explicitly construct the string here (and below)?  The compiler should recognize that `std::string` has a constructor for `const char *` and default-construct a string for you.

Reply:
fixed.